OSInAppBrowserLib/RouterAdapters/OSIABApplicationRouterAdapter.swift

```diff
@@ -9,7 +9,7 @@ public protocol OSIABApplicationDelegate: AnyObject {
 
 /// Provide a default implementations that abstracts the options parameter.
 extension OSIABApplicationDelegate {
-    func open(_ url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any] = [:], completionHandler completion: ((Bool) -> Void)?) {
+    public func open(_ url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any] = [:], completionHandler completion: ((Bool) -> Void)?) {
```

Comment:
Just curious, why was this change necessary?

Reply:
It is not a change related to the feature, but because of that: `Method 'open(_:options:completionHandler:)' must be declared public because it matches a requirement in public protocol 'OSIABApplicationDelegate'`